index.js

```diff
@@ -444,13 +444,11 @@ function sourceInput() {
   var path = core.getInput("path");
 
   if (image && path) {
-    core.setFailed("Cannot use both 'image' and 'path' as sources");
+    throw new Error("Cannot use both 'image' and 'path' as sources");
   }
 
   if (!(image || path)) {
-    core.setFailed(
-      "At least one source for scanning needs to be provided. Available options are: image, and path"
-    );
+    throw new Error("At least one source for scanning needs to be provided. Available options are: image, and path");
```

Comment:
(no change needed) I took another look at this (re our Zoom chat the other day) and it makes sense why this was needed. Throwing an error accomplishes failing the job and skipping the remainder code, only because of the try/catch setup here. Another option is to control the flow via an early return, which lets you control the execution separately from deciding the job status if you ever need to. That might be of interest if we want to move away from large try blocks one day 😃

Reply:
That is a good point. I will open a ticket to track this improvement